site/comps/BlogEntry.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import View, { ViewProps } from "comps/View"
+import { themes } from "lib/ui"
+import Text from "comps/Text"
+import Ctx from "lib/Ctx"
+
+interface BlogEntryProps {
+	title: string;
+	author: string;
+	date: string;
+	description: string;
+	image: string;
+}
+
+const MOBILE = 640
+
+const BlogEntry: React.FC<BlogEntryProps & ViewProps> = ({
+	children,
+	title,
+	author,
+	date,
+	description,
+	image,
+	...args
+}) => {
+	return (
+		<View stretchX={true} height={180} dir="row" outlined={true} rounded={true} bg={3} padX={1} padY={1} css={{
```

Comment:
If a prop value is `true` you can omit it, like `<View stretchX rounded outlined></View>`